compiler/src/iree/compiler/GlobalOptimization/Passes.cpp

```diff
@@ -23,6 +23,10 @@ static llvm::cl::opt<bool> clEnableQuantizedMatmulReassociation(
     llvm::cl::desc(
         "Enables reassociation of quantized matmul ops (experimental)."),
     llvm::cl::init(false));
+static llvm::cl::opt<bool>
+    clGeneralizeLinalgMatmulOps("enable-generalize-linalg-matmul-ops",
```

Comment:
This needs to be codified somewhere, but there is some thoughts about adding `-experimental` to flag names that are not really intended to be user facing (i.e. papering over backend issues that are WIP), per this discussion: https://github.com/iree-org/iree/issues/17788#issuecomment-2218689602. Maybe something like this, but open to suggestions.
```suggestion
    clGeneralizeLinalgMatmulOps("iree-global-opt-generalize-linalg-matmul-ops-experimental",
```

Reply:
Thats a good idea, added that to the name